OpenRA.Mods.Common/Traits/Air/Aircraft.cs

```diff
@@ -49,8 +49,12 @@ public class AircraftInfo : PausableConditionalTraitInfo, IPositionableInfo, IFa
 
 		public readonly int InitialFacing = 0;
 
+		[Desc("How fast can this actor change its body facing?")]
 		public readonly int TurnSpeed = 255;
 
+		[Desc("How fast can this actor change its flight facing? defaults to TurnSpeed when -1. This parameter only applies to aircraft with CanSlide.")]
+		public readonly int FlightTurnSpeed = 255;
```

Comment:
Shouldn't this default to `TurnSpeed` (i.e. `-1`)?

Reply:
Defaulting to 255 regardless of the value of `TurnSpeed` means that the old sliding behaviour (or a close enough approximation of it) remains intact if `FlightTurnSpeed` isn't touched.